.editorconfig

```diff
@@ -4,4 +4,16 @@ root = true
 [*]
 indent_style = tab
 end_of_line = lf
-charset = utf-8
\ No newline at end of file
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.json]
+indent_style = space
```

Comment:
This should be tabs: https://github.com/balena-io/process/blob/master/process/TypeScript_Coding_Guide.md#indentation and not spaces. What was followed to add this instead?

Reply:
Heya @hedss, this rule targets only *.json files, not typescript. (This aligns with json indentation across all components). Should we switch package.json and package.lock.json to tabs indentation (maybe everywhere)?

Reply:
@mikesimos Sorry, yes, you're absolutely right, and it should stick as it is!

Reply:
No worries man! And thanks for checking this :)